### PR TITLE
DX-193, DX-216 - updated info and CMS-handling sections

### PR DESCRIPTION
**Pull request by sushmangupta**
Tickets related - [DX-193](https://shopware.atlassian.net/browse/DX-193), [DX-216](https://shopware.atlassian.net/browse/DX-216)

### CODE REVIEW

File: docs/guides/quick-start/01-introduction.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
+---
+stoplight-id: twpxvnspkg3yu
+---
+
+# Quick Start Guide
+
+## Overview
+
+Shopware Admin API allows web services to perform administrative tasks. This guide is designed to help you understand the Admin API, its endpoints, and its usage. You will get to learn everything about this API.
```
Comment (elkmod) [inline docs/guides/quick-start/01-introduction.md:9]:
```suggestion
The Shopware Admin API allows web services to perform administrative tasks. This guide is designed to help you understand the Admin API, its endpoints, and its usage. You will get to learn everything about this API.
```

File: docs/guides/quick-start/01-introduction.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+    * Assignment of layout to a categoty
+
+It is useful to use an API client like Postman or Insomnia to follow this guide.
+
+## General
+
+The Admin API has a base route that is always relative to your Shopware instance host. Note that it might differ from your sales channel domain. Let us assume your Shopware host is
+
+https://shop.example.com/
```
Comment (elkmod) [inline docs/guides/quick-start/01-introduction.md:53]: Code block? Or does that conflict with our guidelines?

File: docs/guides/quick-start/01-introduction.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+then your Admin API base route will be
+
+https://shop.example.com/api/
+
+The Admin API offers a variety of functionalities referred to as endpoints or nodes, where each has its own route. Refer to [endpoint structure](https://shopware.stoplight.io/docs/admin-api/ZG9jOjEyMzA1ODA5-endpoint-structure) for more details. The endpoints mentioned subsequently are always relative to the API base route.
+
+## Authentication and Setup 
+
+All REST Admin API queries require a valid Shopware access token.The Admin API uses the OAuth 2.0 standard to authenticate users. OAuth 2.0 requires you to obtain an access token to confirm your identity for every server request.
```
Comment (elkmod) [inline docs/guides/quick-start/01-introduction.md:63]:
```suggestion
All REST Admin API queries require a valid Shopware access token. The Admin API uses the OAuth 2.0 standard to authenticate users. OAuth 2.0 requires you to obtain an access token to confirm your identity for every server request.
```

File: docs/guides/quick-start/01-introduction.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+  "body":{
+    "client_id": "administration",
+    "grant_type": "password",
+    "scopes": "write",
+    "username": "your.username@shopware.com",
+    "password": "your.password@123"
+}
+}
+```
+Once you have obtained the access token, simply add it to `Authorization` header as a Bearer token.
```
Comment (elkmod) [inline docs/guides/quick-start/01-introduction.md:88]:
```suggestion

Once you have obtained the access token, simply add it to `Authorization` header as a Bearer token.
```

File: docs/guides/quick-start/06-cms-handling.md [hunk 5]
```diff
@@ -0,0 +1,227 @@
+---
+stoplight-id: s92t5mvj387wz
+---
+
+# CMS
+
+CMS system is referred to as "Shopping Experiences" built upon pages which can be reused and dynamically hydrated based on their assignments to categories or other entities.
```
Comment (elkmod) [inline docs/guides/quick-start/06-cms-handling.md:7]:
```suggestion
Shopware's built-in CMS system is referred to as "Shopping Experiences". It is built upon layouts which can be reused and dynamically hydrated based on their assignments to categories or other entities.
```

File: docs/guides/quick-start/06-cms-handling.md [hunk 6]
```diff
@@ -0,0 +1,227 @@
...
+
+Using CMS REST API, you can perform CRUD operations on section, slot, block and page templates.
```
Comment (elkmod) [inline docs/guides/quick-start/06-cms-handling.md:9]:
```suggestion
Using the CMS REST API, you can perform CRUD operations on sections, slots, blocks and page templates.
```

**Review by elkmod — CHANGES_REQUESTED** [review]

File: docs/guides/quick-start/01-introduction.md (hunk 2 above)
Comment (sushmangupta) [reply docs/guides/quick-start/01-introduction.md:54]: We can use code block here. I have updated it

**Pull request merged**